static/js/models/database.js

```diff
@@ -19,7 +19,7 @@ function handleDisconnect() {
 
     console.log("We ran handleDisconnect...")
     connection.connect(function(err) {
-        console.log("There was an error connection")
+        console.log("There was an error connecting")
```

Comment:
we should actually remove this line. It was there for me testing but I forgot to remove it